### PR TITLE
Added to view corrected answer after selected

### PR DESCRIPTION
**Pull request by lidyaT21**
1. .A user can see what the correct answer is after they selected their answer.
 I modified the answer list  if the user selected wrong answer the app tell the correct answer through green color

2. I have added some feature to the app based on our discussion it just to aesthetic but this will be decide together. just an idea

### CODE REVIEW

File: public/background.jpg
Comment (imtiaz101325) [inline public/background.jpg:1]: You could probably find a better image without the watermark.

File: public/style.css
Comment (imtiaz101325) [inline public/style.css:1]: It looks like you are removing CSS variables and hard-coding color code. It's more maintainable to have these values in a variable.

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -21,48 +21,54 @@ export const initQuestionPage = () => {
   answersListElement.classList.add('answers-list');
 
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
-    const answerElement = createAnswerElement(key, answerText);
-    // Answers options look little bit better now
-    answerElement.classList.add('answer-element');
-    answersListElement.appendChild(answerElement);
+    // Create a button for each answer option
+    const answerButton = document.createElement('button');
```
Comment (imtiaz101325) [inline src/pages/questionPage.js]: This should be done inside reusable view functions.

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -21,48 +21,54 @@ export const initQuestionPage = () => {
...
 
       // Selected answer receives a color depending on whether it's correct or not.
       // Then user moves to the next question
       const answerCorrect = function () {
-        answerElement.classList.add('answer-correct');
-        setTimeout(function () {
-          nextQuestion();
-        }, 500);
+        answerButton.classList.add('answer-correct');
+        setTimeout(function () {}, 500);
```
Comment (imtiaz101325) [inline src/pages/questionPage.js]: Since you are not doing anything here, it's better to remove the `setTimeout`s.

**Review by imtiaz101325 — COMMENTED** [review]
Overall great effort but there seems to be some overlap between your and @xrustik666's work. Best you discuss these changes.

**Review by xrustiks — APPROVED** [review]

**Pull request merged**